docs/docs/plugin-authoring.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Plugin Authoring
+---
+
+Creating custom plugins in Gatsby is a breeze with our straightforward authoring process.
+
+## Core Concepts
+
+- Every Gatsby plugin is a standalone npm package
+- At minimum, a `package.json` is required
+- Plugins can be used locally (see [local plugins](#local-plugins) below) or published to npm as packages
```

Comment:
Does this concept conflict with the first concept "Every Gatsby plugin is a standalone npm package"?

Maybe the first concept could be something like "A Gastby plugin is a set of files that can modify Gatsby's behaviour"?